sample/build.gradle

```diff
@@ -109,6 +109,12 @@ android {
                 println it
             }
         }
+
+        task dumpActivityStack(type: com.novoda.gradle.command.ActivityStack) << {
+            getActivityRecords().each {
+                logger.info it.toString()
```

Comment:
No, any ideas what to do?
Am 13.07.2015 16:24 schrieb "Volker Leck" <notifications@github.com>:

> In sample/build.gradle
> <https://github.com/novoda/gradle-android-command-plugin/pull/85#discussion_r34467557>
> :
>
> > @@ -109,6 +109,12 @@ android {
> >                  println it
> >              }
> >          }
> > +
> > +        task dumpActivityStack(type: com.novoda.gradle.command.ActivityStack) << {
> > +            getActivityRecords().each {
> > +                logger.info it.toString()
>
> Does that print anything when run without specifying --info?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/novoda/gradle-android-command-plugin/pull/85/files#r34467557>
> .
>